app/controlplane/pkg/biz/workflowrun.go

```diff
@@ -200,6 +200,12 @@ func (uc *WorkflowRunUseCase) Create(ctx context.Context, opts *WorkflowRunCreat
 	}
 	contractRevision := opts.ContractRevision
 
+	if opts.ProjectVersion != "" {
+		if err := ValidateVersion(opts.ProjectVersion); err != nil {
+			return nil, err
```

Comment:
Shall we wrap the error?

Reply:
I didn't want to wrap the error because it's already a validation error, and I want to keep it clean. But let me add it, tbh I have not wrap my head around on a convention for this... thanks for pointing it out